### ISSUE TITLE
Page <title> string may contain HTML elements

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/exhibits#2478]
### Current behavior

The page `<title>` string may contain "stringified" HTML elements, which are read out loud by screen readers.
```
<title>
"Resource type: Archive/Manuscript / Date Range: <span class="from" data-blrl-begin="500">500</span> to <span class="to" data-blrl-end="1500">1500</span> - Parker Library On the Web - Spotlight at Stanford Search Results"
</title>
```
### Steps to reproduce

Example page: [Parker Library - Date Range: 500-1500](https://exhibits.stanford.edu/parker/catalog?f%5Bformat_main_ssim%5D%5B%5D=Archive%2FManuscript&search_field=manuscript_number&sort=title_sort+asc%2C+pub_year_isi+desc&exhibit_id=parker&range%5Bpub_year_tisim%5D%5Bbegin%5D=500&range%5Bpub_year_tisim%5D%5Bend%5D=1500
).

Select a Date Range from the _Limit your search_ filters. Inspect the resulting page `<title>`.

### Reported by SODA
> Violation: Provide an informative context-sensitive page title
> Severity: Major
>
> [Issue]
> Depending on options chosed, the page title may have HTML inside it. This could be confusing for people. 
> 
> [User Impact]
> When the page <title> of the document does not describe the topic or purpose of the page, users of assistive technology may have trouble finding content or orient themselves.
> 
> [Code Reference]
> `<title>Date Range: <span class="from" data-blrl-begin="500">500</span> to <span class="to" data-blrl-end="1799">1799</span> - Parker Library On the Web - Spotlight at Stanford Search Results</title>`
> 
> [Recommendation]
> Ensure that the value for the <title> element is updated as the page content changes and describes the page's purpose or topic in a concise matter.
